### PR TITLE
Emit all issues upon connecting

### PR DESCRIPTION
**Pull request by sklirg**
~Doesn't currently count votes. If anyone wants a go, feel free.~

Should now count votes correctly.

Here are some issues shown after this change (note: none of these changes implement this list or the design of this list, but it makes them actually render since there are multiple issues in the state):

screenshot 2017-04-18 00 36 36

### CODE REVIEW

File: client/src/components/ConcludedIssue.js [hunk 1]
```diff
@@ -68,17 +68,24 @@ class ConcludedIssue extends React.Component {
   }
 }
 
+ConcludedIssue.defaultProps = {
+  alternatives: [],
+  votes: {},
+  voteDemand: 0,
+  text: 'noCase',
```
Comment (duvholt) [inline client/src/components/ConcludedIssue.js]: :question:

**Comment by sklirg** [comment]
Will have a look at this once more, React didn't get the values on the
initial render.

File: client/src/components/ConcludedIssue.js [hunk 2]
```diff
@@ -33,6 +33,16 @@ class ConcludedIssue extends React.Component {
     };
   }
 
+  componentWillReceiveProps(nextProps) {
+    this.setState({
+      majority: ConcludedIssue.calculateMajority(
```
Comment (duvholt) [inline client/src/components/ConcludedIssue.js]: I know this isn't really a part of this PR, but IMO this should not be a state. 
A state that can be derived from other states/props is not needed.

I think the whole component can be coverted to a pure component and majority can be calculated when rendering.

Comment (sklirg) [inline client/src/components/ConcludedIssue.js]: Sure. As you mentioned, that isn't related to this PR, so I'll create an issue for it instead.

**Pull request merged**